src/components/CallToAction.jsx

```diff
@@ -23,10 +23,10 @@ export function CallToAction() {
           </p>
           <div className='mt-10 flex justify-center gap-x-6'>
             <Button href='https://discord.com/invite/nNtVfKddDD' target='_blank' color='white'>
-              <span className='text-blue-600'>Join us on Discord</span>
+              <span className='text-[#7289DA]'>Join us on Discord</span>
             </Button>
             <Button href='https://www.twitch.tv/ykdojo' target='_blank' color='white'>
-              <span className='text-purple-600'>Join us on Twitch</span>
+              <span className='text-[#9146FF]'>Join us on Twitch</span>
```

Comment:
If you want to make this change, can you send this as a separate PR?

Reply:
You can exclude it, it's fine. It's too minor of a change to notice anyways.